docker/lnd/entrypoint.sh

```
@@ -0,0 +1,29 @@
+# ensures that the files can be read/write from both sides
+if ! id lnd > /dev/null 2>&1; then
+  USERID=${USERID:-1000}
+  GROUPID=${GROUPID:-1000}
+
+  echo "adding user lnd ($USERID:$GROUPID)"
+  groupadd -f -g $GROUPID lnd
+  useradd -M -u $USERID -g $GROUPID lnd
+  chown -R $USERID:$GROUPID /home/lnd
+fi
+
+if [ $(echo "$1" | cut -c1) = "-" ]; then
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)